src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -176,13 +176,18 @@ def get_sync_policy(self, bucket_name='', zonegroup_name='', all_policy=None):
         if all_policy:
             sync_policy_list = []
             buckets = json.loads(RgwBucket().list(stats=False))
+            default_zonegroup = ''
+            for zonegroup in RgwMultisite().get_all_zonegroups_info()['zonegroups']:
+                if zonegroup['id'] == RgwMultisite().get_all_zonegroups_info()['default_zonegroup']:
+                    default_zonegroup = zonegroup['name']
```

Comment:
Just a concern, are we ensured that these lookups wont results in `KeyError`

Reply:
added more checks